### DISCUSSION TITLE
Running multiple tasks concurrently, which are using the same task template within the same project.

### DISCUSSION THREAD

**Discussion by karlmjogila** [semaphoreui/semaphore#1528] [Q&A]
Let's say I have a bootstrap task template and I want to run multiple instances of that within the same project concurrently, but targeting different host groups. Is that even possible?

**Comment by lraphael** [comment]
I encountered the same restriction yesterday. I had a task to start an installation with different variables, and each task took about 40 minutes. Since I needed four of them, it took a while. 

I have seen this [pull request ](https://github.com/ansible-semaphore/semaphore/pull/1000), and if it does not cause problems, I may make my own version without this limitation.

For the moment, I have copied the template four times, but it is more of an interim solution because the maintenance effort increases and the overview decreases.

**Comment by MSandro** [comment]
any news here?
seems it still does not work.
